dump/plugins/dash_eni.py

```diff
@@ -0,0 +1,63 @@
+from dump.helper import create_template_dict
+from dump.match_infra import MatchRequest
+from swsscommon.swsscommon import SonicDBConfig
+from dash_api.eni_pb2 import Eni
+from dash_api.vnet_pb2 import Vnet
+from .executor import Executor
+
+
+APPL_DB_SEPARATOR = SonicDBConfig.getSeparator("APPL_DB")
+
+
+class Dash_Eni(Executor):
+    """
+    Debug Dump Plugin for DASH VNET Mapping
+    """
+    ARG_NAME = "dash_eni_value"
+
+    def __init__(self, match_engine=None):
+        super().__init__(match_engine)
+        self.is_dash_object = True
+
+    def get_all_args(self, ns=""):
+        req = MatchRequest(db="APPL_DB", table="DASH_ENI_TABLE", key_pattern="*", ns=ns)
+        ret = self.match_engine.fetch(req)
+        appliance_tables = ret["keys"]
+        return [key.split(APPL_DB_SEPARATOR)[-1] for key in appliance_tables]
+
+    def execute(self, params):
+        self.ret_temp = create_template_dict(dbs=["APPL_DB", "ASIC_DB"])
+        dash_eni_table_name = params[self.ARG_NAME]
+        self.ns = params["namespace"]
+        self.init_dash_eni_table_appl_info(dash_eni_table_name)
+        self.init_dash_eni_table_asic_info(dash_eni_table_name)
+        return self.ret_temp
+
+    def init_dash_eni_table_appl_info(self, dash_eni_table_name):
+        req = MatchRequest(db="APPL_DB", table="DASH_ENI_TABLE", key_pattern=dash_eni_table_name, ns=self.ns,)
+        ret = self.match_engine.fetch(req)
+        self.add_to_ret_template(req.table, req.db, ret["keys"], ret["error"])
+
+    def init_dash_eni_table_asic_info(self, dash_eni_table_name):
+        req = MatchRequest(db="APPL_DB", table="DASH_ENI_TABLE", key_pattern=dash_eni_table_name, return_fields=["vnet"], ns=self.ns, pb=Eni())
```

Comment:
Why call DB twice for DASH_ENI_TABLE? Can't we save vnet in the method "init_dash_eni_table_appl_info"